src/pages/account/profile/index.tsx

```diff
@@ -27,27 +27,36 @@ import { sentryMiddleware } from "@src/middlewares/sentry-middleware";
 const ProfilePage: React.FC = () => {
   const router = useRouter();
 
-  const { data, error } = useSWR<
-    {
-      userProfile: Profile;
-      userAddresses: Address[];
-    },
+  const { data: userProfile, error: userProfileError } = useSWR<
+    Profile,
     SWRError
-  >(`/api/profile/get-user-profile-and-addresses`);
+  >(`/api/profile/user-profile`);
 
-  if (error) {
-    if (error.statusCode === HttpStatus.NOT_FOUND) {
+  if (userProfileError) {
+    if (userProfileError.statusCode === HttpStatus.NOT_FOUND) {
       router && router.push("/account/profile/user-profile/new");
     }
   }
```

Comment:
I don't think we should redirect to the page automatically and replace it with a button to create a Profile maybe?
Wdyt?

Reply:
That's a real question indeed. We said we would do that way in the first place, but that could be great to discuss that again with the team. Personally, I kinda like that way, because it feels "automatic", less click to perform an action, but I'm not close to add a button instead 🙂

Reply:
I like it both ways and as you said, we should discuss it 🙂
(Can you create a task about that?)

Reply:
Yep, I will create it. 👍